src/wasm.h

```diff
@@ -90,6 +90,9 @@ struct Name : public cashew::IString {
 
 // An index in a wasm module
 typedef uint32_t Index;
+inline Index toIndex(size_t index) {
```

Comment:
Is the plan to use this in more places? If not, let's just put a cast where the one use is. If so (and especially if there will be a lot) it might be cleaner to do something with an implicit conversion like we did with Address. (that can be a different PR).